src/applications/find-forms/components/SearchResult.jsx

```diff
@@ -60,7 +60,7 @@ const SearchResult = ({ form }) => {
 
       <dd className="vads-u-padding-bottom--3">
         <a
-          download={form.attributes.url}
+          download={`${form.id}.pdf`}
```

Comment:
Will the form download always be a `pdf`? Otherwise, I may want to derive the file extension from `form.attributes.url` + regex

Reply:
Good point, there are a couple of non-PDF links. Maybe `download` should be dynamic then, only passed as a prop if the form's URL ends with `.pdf`.